### ISSUE TITLE
[Question]

### ISSUE THREAD

**Issue by Sobhi-Elbsha** [OpenBoard-org/OpenBoard#1024]
can you put feature to openboard like draw triangle or rectangle to make this app the best

**Comment by letsfindaway** [comment]
I don't think that such features will arrive in OpenBoard, soon. They contradict the idea that OpenBoard is an electronic analog of a blackboard.

See e.g. the compass tool for drawing a circle. You not just use the mouse to draw a circle on the board. Instead you use the compass, position the pin, adjust the radius and then move the pen around, just like you would do it with a real compass. The same is for other shapes. You have a ruler, a triangle and a protractor, just like the real tools the students use.

My wife is a teacher using OpenBoard for distance learning. More than once she tells me how impressive and at the same time obvious it is to draw a circle in this way. The same idea of "do it like the students do" applies also to triangle and rectangle.

**Comment by Vekhir** [comment]
FWIW, on the right panel, there's a folder named `Shapes` which contains among other things rectangles and triangles which you can stretch and move around. They aren't perfect, but good enough for rough sketches. If you need precision, the various tools mentioned above work wonders too.
